build/apply-certs.sh

```diff
@@ -1,18 +1,22 @@
 #!/bin/bash
 
 if [ "$1" == "" ]; then
-  echo "usage: ./apply-certs.sh <cluster_context>"
+  echo "usage: ./apply-certs.sh <cluster_context> <namespace>"
   exit 1
 fi
 
+if [ "$2" == "" ]; then
+  echo "usage: ./apply-certs.sh <cluster_context> <namespace>"
```

Comment:
If namespace is an optional argument to make-certs, it should be an optional argument here too (and both should use the same default value)

Reply:
Opting to make it required throughout per `make-certs.py` comment.